### PR TITLE
Civ 1525 redis claim service

### PR DESCRIPTION
**Pull request by TatsyMaslova**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/modules/draft-store/draftStoreService.ts [hunk 1]
```diff
@@ -22,18 +24,26 @@ export class DraftStoreService {
...
+    draftStoreClient.set(claimId, JSON.stringify(storedClaimResponse));
   }
 
   private createNewCivilClaimResponse(claimId: string) {
     const storedClaimResponse = new CivilClaimResponse();
     storedClaimResponse.id = claimId;
     return storedClaimResponse;
   }
+
+  private convertRedisData(data:any): CivilClaimResponse{
```
Comment (LeonardoPalmeiro) [inline src/main/modules/draft-store/draftStoreService.ts:42]: that we have in Redis is String of representation of json, might be better change data:any to data: string,

Comment (LeonardoPalmeiro) [reply src/main/modules/draft-store/draftStoreService.ts:42]: maybe the best name for this method could be convertRedisDataToCivilClaimResponse or something like this. what do you think about it?

File: src/main/modules/draft-store/draftStoreService.ts [hunk 2]
```diff
@@ -22,18 +24,26 @@ export class DraftStoreService {
...
+    let jsonData = undefined;
+    if(data){
+      jsonData = JSON.parse(data);
+    }
+    return jsonData? Object.assign(jsonData, new CivilClaimResponse()): undefined;
```
Comment (LeonardoPalmeiro) [inline src/main/modules/draft-store/draftStoreService.ts:47]: I believe that jsonData will be always null in this case. 

we need all methods on from claims class like formattedResponseDeadline formattedTotalClaimAmount, in this case my suggestion is 
    return Object.assign(new CivilClaimResponse(), jsonData); 
in this case if we don't have the jsonData will create a empty object of CivilClaimResponse.

**Review by LeonardoPalmeiro — CHANGES_REQUESTED** [review]

File: src/main/modules/draft-store/draftStoreService.ts (hunk 1 above)
Comment (TatsyMaslova) [reply src/main/modules/draft-store/draftStoreService.ts]: done 👍

**Review by laarmada — APPROVED** [review]
Amazing!

**Review by LeonardoPalmeiro — APPROVED** [review]
great!!

**Pull request merged**